static/src/deploys-radiator/app/render.js

```diff
@@ -0,0 +1,124 @@
+        return h('li', {
+            className: `deploy deploy--${deployGroup.status.split(' ').join('-').toLowerCase()}`
+        }, [
+            h('h2', [
+                h('a', {
+                    href: createBuildLink(deployGroup.build)
+                }, `${deployGroup.build}`)
+            ]),
+            // Only show project names if we have multiple deployed groups
+            exp(shouldShowProjectNames) && ih('ul', {}, groupDeploys
+                .sortBy(build => build.projectName)
+                .map(deploy => {
+                    const previousBuild = previousDeploysMap.get(deploy);
+                    return h('li', [
+                        h('a', {
+                            href: createRiffRaffDeployLink(deploy.uuid),
+                            title: previousBuild ? `Previous build: ${previousBuild.build}` : ''
+                        }, deploy.projectName)
+                    ]);
+                })
+                .toList())
+        ]);
+    };
+    const createDeployGroup = (deploys) => (deploys.groupBy(deploy => (createDeployGroupRecord({
+        status: deploy.status,
+        build: deploy.build
+    }))));
+    const currentDeployGroupNodes = createDeployGroup(currentDeploys)
+        .map(renderGroupDeployNodes)
+        .toList();
+    const notRunningDeploysGroup = createDeployGroup(notRunningDeploys);
+    return h('div', {}, [
+        ih('ul', {
+            className: 'deploys'
+        }, currentDeployGroupNodes),
+        exp(notRunningDeploys.size > 0) && [
+            h('h2', 'Queue'),
+            ih('ul', {}, notRunningDeploysGroup
+                .map((groupDeploys, deployGroup) => (h('li', [
+                    h('strong', [
+                        h('a', {
+                            href: createBuildLink(deployGroup.build)
+                        }, deployGroup.build.toString())
+                    ]),
+                    ih('ul', {}, (groupDeploys
+                        .sortBy(build => build.projectName)
+                        .map(deploy => {
+                            const previousBuild = previousDeploysMap.get(deploy);
+                            return h('li', [
+                                h('a', {
+                                    href: createRiffRaffDeployLink(deploy.uuid),
+                                    title: previousBuild ? `Previous build: ${previousBuild.build}` : ''
+                                }, deploy.projectName)
+                            ]);
+                        })
+                        .toList()))
+                ])))
+                .toList())
+        ]
+    ]);
+};
+const renderPage = ([codeDeploys, prodDeploys], [latestCodeDeploy, oldestProdDeploy], commits) => {
+    const isInSync = oldestProdDeploy.build === latestCodeDeploy.build;
+    return h('.row#root', {}, [
+        h('h1', [
+            `Status: ${isInSync ? 'in sync. Ship it!' : 'out of sync.'}`
```

Comment:
I always wondered why it says `Ship it` when it's in sync, doesn't that mean there's nothing to ship?